examples/05_glm_second_level/plot_second_level_one_sample_test.py

```diff
@@ -187,37 +191,33 @@
 
 IMAGES = [
     neg_log_pval,
-    out_dict['logp_max_t'],
-    out_dict['logp_max_size'],
-    out_dict['logp_max_mass'],
+    out_dict["logp_max_t"],
+    out_dict["logp_max_size"],
+    out_dict["logp_max_mass"],
 ]
 TITLES = [
-    'Parametric Test',
-    'Permutation Test\n(Voxel-Level Error Control)',
-    'Permutation Test\n(Cluster-Size Error Control)',
-    'Permutation Test\n(Cluster-Mass Error Control)',
+    "Parametric Test",
+    "Permutation Test\n(Voxel-Level Error Control)",
+    "Permutation Test\n(Cluster-Size Error Control)",
+    "Permutation Test\n(Cluster-Mass Error Control)",
 ]
 
 fig, axes = plt.subplots(figsize=(8, 8), nrows=2, ncols=2)
-img_counter = 0
```

Comment:
itertool was introduced through the refactoring of the nested loop

Reply:
Right ! Thanks !
Thank god it's Friday... 😅